.cirrus.yml

```diff
@@ -1,6 +1,6 @@
 env:
   PATH: "$HOME/.cargo/bin:$PATH"
-  RUST_VERSION: '1.70.0' # Needs to be <= FreeBSD version
```

Comment:
Don't think the Rust version needs to change for this.

Reply:
Yes please keep it as-is (1.70.0). The idea is that we test with a minimum supported Rust version. But since rustup doesn't have versions for FreeBSD we just use whatever version is packaged there. The comment notes that we should never use a newer version than what is packaged on FreeBSD for the platforms that do use rustup.